roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1646,6 +1646,8 @@ def get_defaults(self, roles, deployment_type):
                     deploy=False,
                     storage=dict(
                         kind=None,
+                        duration=7,
```

Comment:
duration and resolution options should be at the same level that deploy or storage